src/integration/sift/dto/sift.dto.ts

```diff
@@ -1008,6 +1008,9 @@ export const SiftAmlDeclineMap: { [method in AmlReason]: DeclineCategory } = {
   [AmlReason.CHF_ABROAD_TX]: DeclineCategory.INVALID,
   [AmlReason.ASSET_KYC_NEEDED]: DeclineCategory.OTHER,
   [AmlReason.CARD_NAME_MISMATCH]: DeclineCategory.OTHER,
+  [AmlReason.USER_BLOCKED]: DeclineCategory.OTHER,
+  [AmlReason.USER_DATA_BLOCKED]: DeclineCategory.OTHER,
```

Comment:
User blocked is probably `RISKY`?

Reply:
Hm, could also be that we blocked him because his wallet got hacked for example 🤔 don't know if that RISKY Option would lead to problems for this users, but eah we could do that 😅